ui/widgets/draggable.js

```diff
@@ -98,6 +98,10 @@ $.widget( "ui.draggable", $.ui.mouse, {
 		}
 		this._removeHandleClassName();
 		this._mouseDestroy();
+
+		if ( $.ui.ddmanager ) {
```

Comment:
This doesn't seem right. A check for `$.ui.ddmanager.current` being the draggable that's getting destroyed could work, but this implementation is error prone.